roles/iridl/tasks/main.yaml

```diff
@@ -1,18 +1,33 @@
+- name: Include OS-Specific Variables
+  include_vars: "{{ ansible_distribution~ansible_distribution_major_version }}.yaml"
+
+- name: Setup OS-Specific Tasks
+  include_tasks: "{{ ansible_distribution~ansible_distribution_major_version }}_setup.yaml"
+
 #
 # General system settings
 #
 - name: enable EPEL yum repo
-  yum:
+  package:
     name: epel-release
 
-- name: yum
-  yum:
+- name: latest version packages
+  package:
     name:
       - screen
       - tmux
       - htop
       - iotop
+      - ca-certificates
+      - chrony
+    state: latest
```

Comment:
I set these specifically to update because they wouldn't affect the running of the Data Library and not having the latest version of ca-certificates, for example, could actually cause problems.  But you're right, we should probably release a patch every time there's an OS updated and just run a full yum update on the whole system.